collect_app/src/main/java/org/odk/collect/android/tasks/DiskSyncTask.java

```diff
@@ -49,15 +49,25 @@
 
     int instance;
 
-    DiskSyncListener mListener;
+    DiskSyncListener listener;
 
     String statusMessage;
 
-    private FormsDao mFormsDao;
+    private FormsDao formsDao;
+   
+    private static class UriFile {
```

Comment:
Is there a convention for private inner classes? I thought they went at the end.